### ISSUE TITLE
TypeScript compile error for protractor.promise

### ISSUE THREAD

**Issue by skywalker5727** [angular/protractor#3437]
Bug report
---
Converting my tests to TypeScript.  Test uses Protractor promise
    var deferred = protractor.promise.defer();
added 'protractor' to import from protractor/globals.
Getting Typescript error: Property 'promise' does not exist on type 'Protractor'.
Using Visual Studio typescript compiler or gulp-typescript with tsconfig.json
  "compilerOptions": {
    "target": "es2015",
    "module": "commonjs",
    "moduleResolution": "node",
    "sourceMap": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "removeComments": false,
    "noImplicitAny": false,
    "outDir": "./built"

Are there missing typings to use protractor.promise?

- Node Version: `6.3.1`
- Protractor Version: `4.0.2`
- Browser(s): ``
- Operating System and Version `Windows 10`
- Your protractor configuration file
- A relevant example test
- Output from running the test
- Steps to reproduce the bug
- The URL you are running your tests against (if relevant)

Feature Request
---

- Reasons for adopting new feature
- Is this a breaking change? (How will this affect existing functionality)

**Comment by cnishina** [comment]
I believe this is because I created a different interface and applied it to the protractor object. In the global protractor, promise actually exists but then this happened:

globals.ts
```
export interface Protractor {
  // defined some variables that are associated with Protractor
}
export let protractor: Protractor = global['protractor'];
```

This should use the Protractor namespace... will be addressed.

**Comment by cnishina** [comment]
closed with 5f690fe0d0526d5ed4cc482fb5915d28eedbe11e
